externs/freedom_extern.js

```diff
@@ -96,6 +96,14 @@ function FreedomCrypto() {};
  */
 FreedomCrypto.prototype.getRandomBytes = function(numBytes) {}
 
+/**
+ * @typedef {{
+ *   data: !ArrayBuffer,
+ *   signedBy: Array<string>
+ * }}
+ */
+var FreedomPgpDecryptResult;
```

Comment:
It's just for readability, because `FreedomPgpDecryptResult` is clearer than `{data: !ArrayBuffer, signedBy Array<string>}`, and it now appears in two places in `socketio.quiver.js`.